web/js/components/date-selector/date-selector.js

```diff
@@ -1,61 +1,68 @@
 /* eslint-disable react/jsx-props-no-spreading */
```

Comment:
This override is functional, but it's worth considering getting rid of it (wv-2579 intends to address overrides). It appears we just need to eliminate `{...sharedProps}` when passing props in a handful of places.